### PR TITLE
chore: more info logging, noissue

### PR DESCRIPTION
**Pull request by sadams**

### CODE REVIEW

File: .github/workflows/audit.yml [hunk 1]
```diff
@@ -3,11 +3,11 @@ name: NPM Audit
...
-        with:
-          node-version: 12
-      - run: npm audit
+    audit:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v1
+            - uses: actions/setup-node@v1
+              with:
+                  node-version: 12
```
Comment (ikhemissi) [inline .github/workflows/audit.yml:12]: **not part of this change**: we should update our github workflows to test with node 14 instead of 12

File: src/createHandler.js [hunk 2]
```diff
@@ -89,16 +89,15 @@ function createHandler(config) {
...
+			logger.info('Invoking handler for message from queue', {
+				messageMetadata,
+				queueUrl,
+			});
 			try {
 				await invokeHandler(handler, message, messageMetadata);
 			} catch (e) {
 				success = false;
-				logger.error(`Failure handling message: ${e}`);
+				logger.error('Failure handling message', e);
```
Comment (ikhemissi) [inline src/createHandler.js:100]: You can add the error context here (e.g. message metadata) to help with debugging like:
```
logger.error('Failure handling message', {messageMetadata, queueUrl}, e);
```

**Review by ikhemissi — APPROVED** [review]

File: .github/workflows/audit.yml (hunk 1 above)
Comment (sadams) [reply .github/workflows/audit.yml]: done

File: src/createHandler.js (hunk 2 above)
Comment (sadams) [reply src/createHandler.js]: done
